modules/nw-ovn-kubernetes-ansible-migration-about.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-ansible-migration-about_{context}"]
+= Using the Ansible plybook to migrate to the OVN-Kubernetes network plugin
+
+As a cluster administrator, you can use an Ansible playbook to migrate from the OpenShift SDN Container Network Interface (CNI) network plugin to the the OVN-Kubernetes plugin for your cluster.
+
+The Ansible playbook includes the following set of playbooks that each represent a stage in the migration process:
+
+. Prechecks
+. Migrate
+. Reboot
+. Post-migration
+
+[IMPORTANT]
+====
+If you run a playbook as a non-cluster administrator, the playbook fails.
+====
+
+If a playbook fails, check the playbook output for the source of the error. Resolve the error and then rerun the Ansible playbook.
+
+.Prerequisites
+
+* You set a bastion server.
+// LINK for this packages in a RH resource?
```

Comment:
probably refer to ansible installation documentation?

Reply:
HI @gsr-shanks . Would that be the following document?


https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.5/html-single/getting_started_with_playbooks/index#ref-how-to-use-ansible-playbooks

The installation docs do not mention an Ansible package directly.

Reply:
I see:
`$ sudo dnf install ansible`

Reply:
Thanks. Updated.